kaolin/render/spc/raytrace.py

```diff
@@ -48,4 +50,43 @@ def unbatched_raytrace(octree, points, pyramid, exsum, origin, direction, level)
         direction.contiguous(),
         level)
 
+def mark_first_hit(nuggets):
+    r"""Mark the first hit in the nuggets. 
+
+    Returns:
+        first_hits (torch.BoolTensor):
```

Comment:
@tovacinni can you add some details to the descriptions? 
- What does "Mark the first hit" means?
- Can you add some description for the arguments and return ? (shape will be useful too)

Reply:
Done